src/libraries/System.Private.CoreLib/src/System/IO/FileSystemInfo.Unix.cs

```diff
@@ -63,13 +63,19 @@ internal DateTimeOffset LastWriteTimeCore
 
         internal long LengthCore => _fileStatus.GetLength(FullPath);
 
+        internal UnixFileMode UnixFileModeCore
+        {
+            get => _fileStatus.GetUnixFileMode(FullPath);
+            set => _fileStatus.SetUnixFileMode(FullPath, value);
+        }
+
         public void Refresh()
         {
             _linkTargetIsValid = false;
             _fileStatus.RefreshCaches(FullPath);
         }
 
-        internal static void ThrowNotFound(string path)
+        internal static void ThrowNotFound(string? path)
```

Comment:
Is this change still necessary with the latest code? I'm not sure who is passing in a nullable string currently.